src/liballoc/slice.rs

```diff
@@ -140,7 +140,6 @@ mod hack {
     use crate::string::ToString;
     use crate::vec::Vec;
 
-    #[inline]
```

Comment:
Please add a comment explaining that this is *deliberately* non-`inline` and pointing to the perf results in this PR. Otherwise, someone is inadvertently going to add the attribute back in a year or two.

Reply:
That makes sense, will do!